### PR TITLE
Update Terrain Following pages with new utility and info

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-terrain-following.rst [hunk 1]
```diff
@@ -85,29 +85,28 @@ the terrain may vary significantly. Key uses are:
...
-(around 30 meters in the US). Support for other terrain databases can be
-added by extending the ground station code without changes to the
-ArduPilot code.
+
+Sources of Terrain Data
+=======================
+
+The ground station is normally responsible for providing the raw terrain data which is sent to the aircraft via MAVLink. Right now only Mission Planner and MAVProxy support the required TERRAIN_DATA and TERRAIN_REQUEST MAVLink messages needed for terrain following download support. If you are using a different ground station , in order to download terrain data you will need to connect using one of those two ground stations in order to allow ArduPilot to load terrain data onto your board on the ground or in flight.  Once it is loaded, it is saved permanently on the microSD card.
+
+Both MissionPlanner and MAVProxy support the global SRTM database for terrain data. That database has a global grid spacing of 3 arc-seconds (around 100 meters), but has a smaller grid spacing in some parts of the world (around 30 meters in the US).
```
Comment (tridge) [inline common/source/docs/common-terrain-following.rst:94]: our SRTM database only includes the 100m grid. There is a 30m grid available (it is worldwide now I believe), but we don't make it available yet.

File: copter/source/docs/terrain-following.rst [hunk 2]
```diff
@@ -30,33 +30,52 @@ Setting up a Mission to use Terrain data
 
     Do not set the :ref:`EK2_RNG_USE_HGT <EK2_RNG_USE_HGT>`  or :ref:`EK3_RNG_USE_HGT <EK3_RNG_USE_HGT>` parameters.  These parameters should be left at "-1".
 
+Sources of Terrain Data
+=======================
+
+The ground station is normally responsible for providing the raw terrain data which is sent to the aircraft via MAVLink. Right now only Mission Planner and MAVProxy support the required TERRAIN_DATA and TERRAIN_REQUEST MAVLink messages needed for terrain following download support. If you are using a different ground station , in order to download terrain data you will need to connect using one of those two ground stations in order to allow ArduPilot to load terrain data onto your board on the ground or in flight.  Once it is loaded, it is saved permanently on the microSD card.
+
+Both MissionPlanner and MAVProxy support the global SRTM database for terrain data. That database has a global grid spacing of 3 arc-seconds (around 100 meters), but has a smaller grid spacing in some parts of the world (around 30 meters in the US).
```
Comment (tridge) [inline copter/source/docs/terrain-following.rst:38]: same 30m note

File: common/source/docs/common-terrain-following.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-terrain-following.rst:94]: historical....will update

**Comment by stephendade** [comment]
Something I forgot to mention earlier - any user generated terrain (ie the download link) will be available for 24H before it's deleted. After that time user will have to do a new request.

**Comment by Hwurzburg** [comment]
maybe a note on the gui page for the zip download?

**Comment by stephendade** [comment]
> maybe a note on the gui page for the zip download? otherwise, any changes here?

Yeah, that sounds better.

**Comment by Hwurzburg** [comment]
thanks...merging

**Comment by Kiwa21** [comment]
@stephendade Is it possible to use a different server that provides better resolution ? (30m?)

Thanks !

**Pull request merged**